201-logic-app-transform-function/azuredeploy.json

```diff
@@ -0,0 +1,49 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "existingFunctionAppName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the function container in which the function has to be created."
+            }
+        },
+        "functionName": {
```

Comment:
With this template we create a new function (for transform capabilities) in the existing function app/function container.

This template expects a already existing function app as input. This is the reason CI is also failing. I verified the templates manually.